trymito.io/pages/blog/[slug].tsx

```diff
@@ -1,88 +1,171 @@
+              <div dangerouslySetInnerHTML={{ __html: props.post.html }}/>
+              <div className={postStyles.share_section}>
+                <a className={postStyles.tweet_button}
+                  href={`https://twitter.com/intent/tweet?text=${encodeURIComponent(props.post.title ?? '')}&url=${encodeURIComponent(currentURL ?? '')}`}>
+                  <TwitterLogo />
+                </a>
+                <a className={postStyles.email_button}
+                  href={`mailto:?subject=${encodeURIComponent(props.post.title ?? '')}&body=${encodeURIComponent(currentURL ?? '')}`}>
+                    <EmailIcon />
+                  </a>
+                <a className={postStyles.linkedin_button}
+                  href={`https://linkedin.com/shareArticle?title=${encodeURIComponent(props.post.title ?? '')}&url=${encodeURIComponent(currentURL ?? '')}`}>
+                    <LinkedinLogo />
+                  </a>
+              </div>
+
+              {/* Suggested Posts */}
+              <div className={postStyles.suggested_posts_section}>
+                <h2>More Like This</h2>
+                <div className={postStyles.suggested_posts_container}>
+                  <Link href='/blog/automating-spreadsheets-with-python-101'>
+                    <div className={postStyles.suggested_post}>
+                      <h4>Automating Spreadsheets with Python 101</h4>
+                      <p>How to tell the difference between a good and bad Python automation target.</p>
+                    </div>
+                  </Link>
+                  <Link href='/blog/10-mistakes-to-look-out-for-when-transitioning-from-excel-to-python'>
+                    <div className={postStyles.suggested_post}>
+                      <h4>10 Mistakes To Look Out For When Transitioning from Excel To Python</h4>
+                      <p>10 Common Mistakes for new programmers transitioning from Excel to Python</p>
+                    </div>
+                  </Link>
+                  <Link href='/blog/quantifying-mitos-impact-on-analyst-python-productivity'>
+                    <div className={postStyles.suggested_post}>
+                      <h4>Research shows Mito speeds up by 400%</h4>
+                      <p>We&apos;re always on the hunt for tools that improve our efficiency at work. Tools that let us accomplish more with less time, money, and resources.</p>
+                    </div>
+                  </Link>
+                  <Link href='/blog/choosing-between-sql-and-python-best-practices-for-data-analytics-workflows'>
+                    <div className={postStyles.suggested_post}>
+                      <h4>3 Rules for Choosing Between SQL and Python</h4>
+                      <p>Analysts at the world&apos;s top banks are automating their manual Excel work so they can spend less time creating baseline reports, and more time building new analyses that push the company forward.</p>
+                    </div>
+                  </Link>
```

Comment:
For these `Link` tags to work from an SEO perspective, they need to also have a <a> tag inside of them. Check this out -- after hovering over a link in Chrome, look in the bottom left hand corner to see if Chrome recognizes it as a link 


https://github.com/mito-ds/mito/assets/18709905/9ce760ce-afca-4997-8477-709beb047ac8

In the case of these, it does not! 


So we should update this to be implemented like this: 
<img width="461" alt="Screenshot 2024-02-19 at 10 47 16 AM" src="https://github.com/mito-ds/mito/assets/18709905/23707e4d-21ac-4727-bf38-c3f94bc41e44">